<launch>
  <!-- Arguments -->
  <arg name="move_forward_only" default="false"/>
  <arg name="scan_topic" default="/scan_filtered"/>

  

  <!-- move_base -->

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <!-- <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" /> -->
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <rosparam file="$(find dr_phil_hardware)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find dr_phil_hardware)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find dr_phil_hardware)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find dr_phil_hardware)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find dr_phil_hardware)/param/move_base_params.yaml" command="load" />
    <!-- <rosparam file="$(find dr_phil_hardware)/param/dwa_local_planner_params.yaml" command="load" /> -->
    <rosparam file="$(find dr_phil_hardware)/param/teb_local_planner_params.yaml" command="load" />

    <rosparam file="$(find dr_phil_hardware)/param/navfn_global_planner_params.yaml" command="load" />
    <!-- <param name="DWAPlannerROS/min_vel_x" value="0.0" if="$(arg move_forward_only)" /> -->
    <remap from="/scan" to="$(arg scan_topic)"/>
  </node>
  
  
   <!-- Map server -->

</launch>
